src/css/global.css

```diff
@@ -150,6 +150,10 @@ i {
   font-size: 16px;
 }
 
+input[type="range"] {
+  accent-color: hsl(180, 45%, 40%);
```

Comment:
This color is already defined as `var(--accent-2)`, assuming there's no reason not to use it.

Reply:
--accent-2: hsl(180, 50%, 40%); will make the track dark in light mode for chromium browsers

Reply:
Oh, I see, I looked at the value in the wrong file.